openslides_backend/action/relations/relation_manager.py

```diff
@@ -43,6 +43,40 @@ def get_relation_updates(
         self.process_template_fields(model, instance)
 
         relations: RelationUpdates = {}
+        for field_name in instance:
+            if not model.has_field(field_name):
+                continue
+            field = model.get_field(field_name)
+
+            # only relations are handled here
+            if not isinstance(field, BaseRelationField):
+                continue
+            # ignore template fields, we have to do no relation handling there
+            if isinstance(field, BaseTemplateField) and field.is_template_field(
+                field_name
+            ):
+                continue
+
+            handler = SingleRelationHandler(
+                self.datastore,
+                field,
+                field_name,
+                instance,
+            )
+            result = handler.perform()
+            for fqfield, relations_element in result.items():
+                self.process_relation_element(fqfield, relations_element, relations)
+
+        calculated_field_handler_calls = self.handle_calculated_fields_1(
+            instance, action, model
+        )
+        self.apply_relation_updates(relations)
+        self.handle_calculated_fields_2(relations, calculated_field_handler_calls)
+        return relations
+
+    def handle_calculated_fields_1(
```

Comment:
please rename, something like `get_calculated_field_handler_calls` might be more appropriate

Reply:
Renamed,